java/daikon/chicory/PureMethodInfo.java

```diff
@@ -181,7 +181,9 @@ public PureMethodInfo(
     } else if (obj instanceof Byte) {
       return new Runtime.ByteWrap((Byte) obj);
     } else if (obj instanceof Character) {
-      return new Runtime.CharWrap((Character) obj);
+      @SuppressWarnings("signedness:cast.incomparable")
```

Comment:
Can you add a comment why this can be suppressed?

Reply:
added